conda/base/context.py

```diff
@@ -156,11 +156,12 @@ def default_python_validation(value):
 
 def ssl_verify_validation(value):
     if isinstance(value, str):
-        if not isfile(value) and not isdir(value):
+        if not value == "truststore" and not isfile(value) and not isdir(value):
             return (
                 "ssl_verify value '%s' must be a boolean, a path to a "
-                "certificate bundle file, or a path to a directory containing "
-                "certificates of trusted CAs." % value
+                "certificate bundle file, a path to a directory containing "
+                "certificates of trusted CAs, or 'truststore' to use the OS certificate store."
+                % value
```

Comment:
Better to spell out what we mean with "OS".

```suggestion
                "certificate bundle file, a path to a directory containing "
                "certificates of trusted CAs, or 'truststore' to use the "
                "operating system certificate store." % value
```